playwright/README.md

```
@@ -0,0 +1,147 @@
+# Run with visible browser
+npm run playwright:headed
+
+# Debug mode (step through tests)
+npm run playwright:debug
+```
+
+## Architecture
+
+This framework follows the **Page Object Model (POM)** pattern:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)